README.md

```diff
@@ -270,6 +270,36 @@ prints:
 +-----------+------+------------+-----------------+
 ```
 
+#### Filtering your table
+
+You can make sure that your tables are filtered by giving `get_string` a `row_filter`
+keyword argument, which must a function with one argument `row` returning a bool value.
+The `row`is the list of fields in a row.
+
+For example, to print the example table we built earlier of Australian capital city
+data, so that cities with a population with more than 1000000, we can do this:
```

Comment:
```suggestion
You can make sure that your tables are filtered by giving `get_string` a `row_filter`
keyword argument, which must be a function with one argument `row` returning a Boolean value.
The `row` is the list of fields in a row.

For example, to print the example table we built earlier of Australian capital city
data, so that cities with a population with more than 1,000,000, we can do this:
```